packages/amplify-graphql-auth-transformer/src/resolvers/helpers.ts

```diff
@@ -251,6 +251,27 @@ export const generateOwnerClaimExpression = (ownerClaim: string, refName: string
   return compoundExpression(expressions);
 };
 
+/**
+ *
+ */
+export const getOwnerClaimReference = (ownerClaim: string, refName: string): string => {
+  const expressions: Expression[] = [];
+  const identityClaims = ownerClaim.split(IDENTITY_CLAIM_DELIMITER);
+  const hasMultiIdentityClaims = identityClaims.length > 1;
+  let ownerRef = refName;
+
+  if (hasMultiIdentityClaims) {
+    identityClaims.forEach((_, idx) => {
+      expressions.push();
```

Comment:
Is anything being done with these expressions? seems like a noop

Reply:
This is the logic to just get the VTL variable reference that contains the final computed identity claim value. I've tried to merge this with the existing code, but that looked clumsy. Though this code is verbose, it is easy to maintain.